src/pages/UserPage/_user_page.scss

```diff
@@ -0,0 +1,116 @@
+@import '../../utils/scss/constants';
+
+.user-page {
+  max-width: 1366px;
+  min-height: 100vh;
+  margin: 0 auto;
+  background-color: $color-blackish-primary;
+
+  &__header{
+    height: 100px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  &__container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__inputs {
+    margin-top: 15px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex-wrap: wrap;
+  }
+
+  &__col1, &__col2, &__col3, &__col4{
+    width: 48%;
+
+
+  }
+
+  &__input-box {
+    display: flex;
+    flex-direction: column;
+    margin: 5px 0;
+    background-color: $color-blackish-one;
+    padding: 10px;
+    border: 0;
+    color: $color-greyish-two;
+    outline: none;
+  }
+
+  .input-container{
+    display: flex;
+
+    .input_col1, {
+      width: 75%;
+      margin-right: 2%;
+    }
+    .input_col2{
+      display: flex;
+      width: 25%;
+      .checkbox{
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        width: 100%;
+      }
+    }
+  }
+
+  label {
+    font-size: 14px;
```

Comment:
for font-size and colors use contants from utils/_constants.scss, check everywhere

Reply:
done